ethereum/yearn/all_vaults.sql

```diff
@@ -0,0 +1,154 @@
+BEGIN;
+
+CREATE TABLE IF NOT EXISTS yearn."yearn_all_vaults" (
```

Comment:
just following styles in other dune tables
```suggestion
CREATE TABLE IF NOT EXISTS yearn.yearn_all_vaults (
```